ResponseDetective/Sources/URLEncodedBodyDeserializer.swift

```diff
@@ -0,0 +1,42 @@
+//
+// URLEncodedBodyDeserializer.swift
+//
+// Copyright © 2016-2017 Netguru Sp. z o.o. All rights reserved.
+// Licensed under the MIT License.
+//
+
+import Foundation
+
+/// Deserializes URL-encoded bodies.
+@objc(RDTURLEncodedBodyDeserializer) public final class URLEncodedBodyDeserializer: NSObject, BodyDeserializer {
+
+	// MARK: BodyDeserializer
+
+	/// Deserializes JSON data into a pretty-printed string.
+	public func deserialize(body: Data) -> String? {
+
+		guard let string = String(data: body, encoding: .utf8) else {
+			return nil
+		}
+
+		if #available(macOS 10.10, tvOS 10.0, *) {
+
+			let stringFromQueryItems: ([URLQueryItem]) -> String = {
+				$0.map { "\($0.name): \($0.value ?? "nil")" }.joined(separator: "\n")
+			}
+
+			if let items = URLComponents(string: string)?.queryItems {
```

Comment:
Compiler is not complaining about it? From Apple docs, `URLComponents` is available from macOS 10.12+ (https://developer.apple.com/reference/foundation/urlcomponents). So line 22 shouldn't be
`if #available(macOS 10.12...`? If the answer is `yes` then remember about `*Spec.swift` file.